modules/otel-collector-components.adoc

```diff
@@ -684,6 +684,42 @@ The Debug exporter prints traces and metrics to the standard output.
 ----
 <1> Verbosity of the debug export: `detailed` or `normal` or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
 
+[id="otlp-http-exporter_{context}"]
+=== OTLP HTTP exporter
+
+This is an exporter that will consistently export spans, metrics and logs depending on the routing_key configured.
+
+.OpenTelemetry Collector custom resource with an enabled OTLP exporter
+[source,yaml]
```

Comment:
We already have the OTLP HTTP exporter: https://docs.openshift.com/container-platform/4.14/observability/otel/otel-configuration-of-otel-collector.html#otlp-http-exporter_otel-configuration-of-otel-collector
@frzifus, :slightly_smiling_face: are you updating the existing feature or introducing another, new feature? If it's a new feature, then what is its name and is it an exporter or extension or etc.?

Reply:
oh thats my fault. Its the LB exporter.